package.json

```diff
@@ -73,12 +80,14 @@
     "eslint-plugin-react-hooks": "^5.2.0",
     "eslint-plugin-react-refresh": "^0.4.19",
     "globals": "^16.0.0",
+    "happy-dom": "^18.0.1",
     "postcss": "^8.5.3",
     "tailwindcss": "3",
     "typescript": "~5.8.3",
     "typescript-eslint": "^8.30.1",
     "vercel": "^42.3.0",
     "vite": "^6.3.5",
-    "vite-plugin-vercel": "^9.0.7"
+    "vite-plugin-vercel": "^9.0.7",
+    "vitest": "^3.2.4"
   }
 }
```

Comment:
```suggestion
}

```

\n

Reply:
My editor removes trailing new lines, is there a reason to have it?

Reply:
hmm. the docs claim that `insert-final-newline` defaults to `true`, and it doesn't seem like you've had this issue on other files. weird.